itez/beneficiary/models.py

```diff
@@ -41,3 +41,31 @@ class District(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class WorkDetail(models.Model):
+    """
+    Include Work Detail properties. 
+    """
+    salary = models.DecimalField( 
```

Comment:
lets change this to `gross_salary`

Reply:
that way we can have methods to compute `monthly_salary`

Reply:
alright, working on it